### ISSUE TITLE
Composer can't install 1.1.2 with magento repository in composer.json

### ISSUE THREAD

**Issue by Cros1404** [paytrail/paytrail-for-adobe-commerce#16]
Composer can't install 1.1.2 if magento repository is in compsoer.json
```
$ composer require paytrail/paytrail-for-adobe-commerce:1.1.2
The "hirak/prestissimo" plugin (installed globally) was skipped because it requires a Plugin API version ("^1.0.0") that does not match your Composer installation ("2.3.0"). You may need to run composer update with the "--no-plugins" option.
./composer.json has been updated
The "hirak/prestissimo" plugin (installed globally) was skipped because it requires a Plugin API version ("^1.0.0") that does not match your Composer installation ("2.3.0"). You may need to run composer update with the "--no-plugins" option.
Running composer update paytrail/paytrail-for-adobe-commerce
Loading composer repositories with package information
In Laminas\DependencyPlugin\DependencyRewriterV2::onPrePoolCreate
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires paytrail/paytrail-for-adobe-commerce 1.1.2, it is satisfiable by paytrail/paytrail-for-adobe-commerce[1.1.2] from composer repo (https://repo.packagist.org) but paytrail/paytrail-for-adobe-commerce[1.0.1, ..., 1.1.0] from composer repo (https://repo.magento.com) has higher repository priority. The packages from the higher priority repository do not match your constraint and are therefore not installable. That repository is canonical so the lower priority repo's packages are not installable. See https://getcomposer.org/repoprio for details and assistance.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I can solve it if I temporarily remove this magento repository from "repositories" in composer.json:
```
    "repositories": {
        "0": {
            "type": "composer",
            "url": "https://repo.magento.com/"
        },
```

**Comment by loueranta-paytrail** [comment]
Hi, we are currently working on a fix and will hopefully release it soon

**Comment by loueranta-paytrail** [comment]
#17 should have fixed this on the latest 1.2.0 release
